src/core/js/device.js

```diff
@@ -69,6 +73,10 @@ define(function(require) {
         return /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
     }
 
+    function getAppleScreenWidth() {
+        return (Math.abs(window.orientation) === 90) ? screen.height : screen.width;
```

Comment:
You mean 270? It appeared to work in all orientations on the iPad I was testing on.